frontend/www/js/omegaup/components/course/Intro.vue

```diff
@@ -87,11 +151,13 @@ interface Statement {
 
 @Component({
   components: {
+    FontAwesomeIcon,
     'omegaup-markdown': omegaup_Markdown,
     'omegaup-radio-switch': omegaup_RadioSwitch,
   },
 })
 export default class CourseIntro extends Vue {
+  @Prop({ default: null }) course!: types.CourseDetails;
```

Comment:
```suggestion
  @Prop({ default: null }) course!: types.CourseDetails | null;
```

si no, el default estaría asignando algo que no permite el tipo. y en teoría L9 jamás podría ser falso.

Reply:
Listo